core/src/G3Timestream.cxx

```diff
@@ -515,12 +521,6 @@ G3Timestream G3Timestream::operator +(double r) const
 	return ret;
 }
 
-static
-G3Timestream operator +(double l, const G3Timestream &r)
-{
-	return r+l;
-}
-
```

Comment:
Was there something wrong with these functions besides that they weren't accessible outside this translation unit? They seem generally useful, and could just have the `static` dropped and declarations added to the header file.

Reply:
Since these functions are all static, they actually aren't being used anywhere.  I think people make explicit copies and use the in-place operators instead.  I suspect these implementations are holdovers from before we let numpy handle all the operator business on that end.

Reply:
I believe so, yes.

Reply:
I've added these to the header so that they're available in C++, but let numpy handle the python side.